doc/source/code-examples/advancedexamples.rst

```diff
@@ -2015,7 +2015,7 @@ to allow calculation of expectation values directly from such samples:
 .. testcode::
 
     from qibo import Circuit, gates
-    from qibo.hamiltonians import Z
+    from qibo.hamiltonians import Z as HZ
```

Comment:
Is this causing some issue in the new test? Another option is 
```suggestion
    from qibo import hamiltonians
```
and then use `hamiltonian = hamiltonians.Z(4)`. It is a bit longer but probably less weird than the `HZ` symbol.

Reply:
You could even do
```suggestion
    from qibo import hamiltonians as hs
```
that's half-way (but maybe still a bit weird).

Reply:
Yeah no, it was just to avoid the confusion with the `Z` symbol that I use below. Any of the two suggestions should be fine, thanks.